index.js

```diff
@@ -12,4 +12,4 @@ require('./lib/locales');
 // locale data registered, and with English set as the default locale. Define
 // the `default` prop for use with other compiled ES6 Modules.
 exports = module.exports = IntlMessageFormat;
-Object.defineProperty(exports, 'default', {value: exports});
+exports['default'] = exports;
```

Comment:
We can also revisit this after looking into and understanding how 6to5/esnext deals (or will deal with) inter-module-system deps.